IndyAV/simulation/indyav_gazebo/launch/swan.launch

```diff
@@ -13,7 +13,7 @@
     <arg name="gui" value="$(arg gui)"/>
   </include>
 
-  <arg if="$(eval world_name == 'new_smyrna')" name="start_yaw_deg" value="-97"/>
+  <arg if="$(eval world_name == 'new_smyrna')" name="start_yaw_deg" value="170"/>
```

Comment:
why did this change? were we going the wrong way around the track?

Reply:
I basically started from scratch with the urdf, and when I rebuilt the car, it had a different starting orientation. I changed it so that the car would be parallel with the road again, like you had it I believe.

Reply:
this should go back to -97 since the base_link has been reorientated